### PR TITLE
Packages: Adding RTL CSS support to the packages CSS

### PR DESCRIPTION
**Pull request by youknowriad**
This PR adds RTL support to the packages CSS which allows us to drop the root package folders for all the packages that expose CSS files.

There's a new Webpack step copying the built CSS files from the packages folder into the build folder used by the WordPress registered styles.

**Testing instructions**

 - Check that the `nux` RTL CSS file is generated properly in `build-style` and copied to the global `build` folder.

### CODE REVIEW

File: bin/packages/build.js [hunk 1]
```diff
@@ -98,8 +99,17 @@ function buildStyle( packagePath ) {
 			.then( ( result ) => callback( null, result ) );
 	};
 
+	const postCSSRTLSync = ( ltrCSS, callback ) => {
```
Comment (gziolo) [inline bin/packages/build.js]: Should we merge #8093 and refactor to use the asynchronous code?

File: webpack.config.js [hunk 2]
```diff
@@ -235,6 +236,13 @@ const config = {
 			'deprecated',
 			'dom-ready',
 		].map( camelCaseDash ) ),
+		new CopyWebpackPlugin(
+			gutenbergPackages.map( ( packageName ) => ( {
```
Comment (gziolo) [inline webpack.config.js]: Nice and clean 👍

File: bin/packages/build.js (hunk 1 above)
Comment (youknowriad) [reply bin/packages/build.js]: I don't really care which one goes first but yeah, we should align

**Comment by gziolo** [comment]
Almost there, in case of `npm run build` - output files aren't minified.

**Review by gziolo — APPROVED** [review]
It works great, many thanks for fixing it.

🎉

**Comment by aduth** [comment]
Do we still need the `WebpackRTLPlugin` in the default configuration after this change? I'm trying to understand how the plugin is being used, because it appears to be dead code ([per documentation](https://github.com/romainberger/webpack-rtl-plugin), requires some complementing text extract plugin).

https://github.com/WordPress/gutenberg/blob/a9f6a1e0eee428831a9482e0d3c36ee9e4004056/webpack.config.js#L50-L54

It's also the sole plugin we use which hasn't been updated for Webpack 4 and would break with the upcoming Webpack 5 release.

>(node:54786) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead

**Comment by youknowriad** [comment]
I think when we merged this PR, we were in a mixed where not all the packages were in the packages folder but I think now it not necessary anymore.

**Comment by aduth** [comment]
Okay, I'll plan to push up a pull request in the next day or so.

**Comment by aduth** [comment]
Just to make sure it's tracked, I created an issue at #15146. I expect to find some time to look at this on Friday, and have assigned myself accordingly.

**Pull request merged**